test/unit/SourceLocator/SourceStubber/PhpStormStubsSourceStubberTest.php

```diff
@@ -155,19 +144,18 @@ private function assertSameClassAttributes(CoreReflectionClass $original, Reflec
         self::assertSame($original->getName(), $stubbed->getName());
 
         $this->assertSameParentClass($original, $stubbed);
-        $this->assertSameInterfaces($original, $stubbed);
+
+        // Needs fix in JetBrains/phpstorm-stubs
```

Comment:
Do we have a issue identifier to link? A full URI to the current stub (absolute, with sha1 to sources) also suffices for posterity

Reply:
No, there are no issues. It's just information it can be checked after other jetbrains/phpstorm-stubs upgrade.